packages/forms-web-app/src/views/interested-party-guide/get-involved-preliminary-meetings.njk

```diff
@@ -5,13 +5,13 @@
 {% endblock %}
```

Comment:
There is a slight difference between the font on the prototype and the page. Do we care about it ?

Reply:
It should be "GDS Transport" which I think it is. If its not "GDS Transport" then needs to be corrected

Reply:
The font is GDS Transport. But please compare [https://applications-prototype.herokuapp.com/mvp/interested-party-guide/having-your-say-examination](https://applications-prototype.herokuapp.com/mvp/interested-party-guide/having-your-say-examination) and [http://localhost:9004/interested-party-guide/have-your-say-examination](http://localhost:9004/interested-party-guide/have-your-say-examination) to see the difference. Also attached screenshot. On the prototype the font seems slightly bigger.
![prototypeVSapp](https://user-images.githubusercontent.com/1644854/151979518-df063734-8304-46e4-a0ef-5674ae4f7218.jpg)

Reply:
For h1 we are using **class="govuk-heading-xl"**  which is there. So I dont see any issue.

Reply:
The font on the page is smaller. have you opened the links to compare ?

Reply:
Yes I checked and can see very minor difference but can't say if prototype is correct.

Reply:
Can we confirm that this will not come back from testing ?